mass_editing/static/description/index.html

```diff
@@ -0,0 +1,68 @@
+<section class="oe_container oe_dark">
```

Comment:
In what section of README.rst, we should add the guideline (index.html content)?